### PR TITLE
Add BEAST3D model: ERayZer with DINOv3 tokenizer, GT cameras, frustum…

### PR DESCRIPTION
**Pull request by PPWangyc**
… 3DGS

Port multi-view BEAST3D into the beast repo as a full-parity ERayZer subclass, following docs/developer_guide.md.

Shared, opt-in changes (erayzer behavior unchanged, verified):
- gaussians_renderer: add camera_frustum_constraint + opt-in frustum_constraint kwarg; capture render_alphas (was discarded) so mask loss has real alphas; accept bg_color=None.
- erayzer_model: add _tokenize_images / _sample_background / _prepare_target subclass hooks to forward; thread frustum_constraint/backgrounds into Renderer; expose render_alphas/pixel_mask outputs.

BEAST3D (beast/models/beast3d/):
- frozen DINOv3 tokenizer with ImageNet normalization (the DinoV3 wrapper omits it)
- GT cameras via _resolve_cameras; pose branch disabled with encoder_n_layer=0
- frustum-constrained render, random-background compositing, masked alpha loss
- registry + config dispatch (Beast3DBeastConfig) + train.py MultiViewDataModule routing with use_mask=True

Config: configs/multiview/beast3d.yaml (reference template). The machine-specific cheese3d fine-tune config (data_dir + init_checkpoint) is kept local and gitignored, alongside erayzer_cheese3d_ft.yaml.

Tests: tests/models/beast3d/ (19 CPU unit tests + 1 gsplat-gated integration test); renderer + base-ERayZer regression tests. ruff + pyright clean.

### CODE REVIEW

File: tests/models/beast3d/test_beast3d_model.py [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+        assert result.pixel_mask is not None
+        assert result.pixel_mask.shape[2] == 1  # single foreground channel
+
+
+# ---------------------------------------------------------------------------
+# TestBeast3DIntegration
+# ---------------------------------------------------------------------------
+
+
+class TestBeast3DIntegration:
```
Comment (themattinthehatt) [inline tests/models/beast3d/test_beast3d_model.py:276]: does this need a `@requires_gsplat_cuda` decorator?

File: tests/models/beast3d/test_beast3d_model.py [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+    fxfycxcy = torch.tensor([40.0, 40.0, 16.0, 16.0]).reshape(1, 1, 4).repeat(b, v, 1)
+    return {'c2w': c2w, 'fxfycxcy': fxfycxcy}
+
+
+# ---------------------------------------------------------------------------
+# TestBeast3DModelConfig
+# ---------------------------------------------------------------------------
+
+
+class TestBeast3DModelConfig:
```
Comment (themattinthehatt) [inline tests/models/beast3d/test_beast3d_model.py]: minor: can you move this to `tests/test_config.py`? (I'll probably change the organization of these config files at some point, but for now I want to keep them all together)

**Review by themattinthehatt — CHANGES_REQUESTED** [review]
@PPWangyc great PR! I left a couple minor comments, after you fix those I'll merge

**Review by themattinthehatt — APPROVED** [review]
